packages/bits/src/lib/toolbar/toolbar.component.spec.ts

```diff
@@ -155,5 +158,36 @@ describe("components >", () => {
                 expect(component.handleSelectionState()).toBeUndefined();
             });
         });
+
+        describe("toolbarKeyboardService >", () => {
+            const keyboardEventMock: KeyboardEvent = {
+                code: KEYBOARD_CODE.ARROW_LEFT,
+                preventDefault: () => {},
+            } as KeyboardEvent;
+
+            it("should call keyboardService 'onKeyDown' method", () => {
+                const spy = spyOn(component["keyboardService"], "onKeyDown");
+
+                component.onKeyDown(keyboardEventMock);
+
+                expect(spy).toHaveBeenCalledWith(keyboardEventMock);
+            });
+
+            it("should call keyboardService 'disableFocusForMoreBtn' and 'initService' on ngAfterViewInit hook", () => {
+                const disableFocusForMoreBtnSpy = spyOn(component["keyboardService"], "disableFocusForMoreBtn");
+                const initServiceSpy = spyOn(component["keyboardService"], "initService");
+
+                component.ngAfterViewInit();
+                expect(disableFocusForMoreBtnSpy).toHaveBeenCalled();
+                expect(initServiceSpy).toHaveBeenCalled();
+            });
+
+            it("should call preventDefault", () => {
```

Comment:
It seems like this test might belong in the spec file for the service since that's where preventDefault is actually getting called from.

Reply:
agree, but here unfortunately, a bunch of imperative works with dom and I firstly decided to test service methods relatively via component. Because inside it I we have access to templete.

Reply:
ok